build.cmd

```diff
@@ -80,6 +82,11 @@ for %%T IN (Restore %MSBuildBuildTarget%, BuildModernVsixPackages) do (
   )
 )
 
+REM Run copy as a final step after all the product components are built
+if /I "%CopyOutputArtifacts%" == "true" (
+  call build\Scripts\CopyOutput.cmd
```

Comment:
Can you put || goto :BuildFailed after this line?

Can you prefix this path relative to the %Root% so that if I run this from another folder it does the right thing?